vivisect/defconfig.py

```diff
@@ -51,13 +52,35 @@
                 'case_failure': 5000,
                 'min_func_instr_size': 10,
                 'timeout_secs': 45,
-            }
+            },
+            'stack':{
+                'base': 0xBFB00000,
+                'mask': 0xFFF00000,
+                'top':  0xBFB08000,
+                'pointer': None,
+            },
+            'taint':{
+                'base': 0x4156000F,
+                'byte': 'a',
+                'offset': 0x1000,
+                'mask': 0xffffe000,
+            },
         },
-        'arch':{
-            'ppc':{
-                'options':'spe'
+
+        'arch': {
+            'ppc': {
+                'options': 'spe',
+                'bootstrap': {
+                    'rchwaddrs': [
+                        0x0000, 0x4000, 0x10000, 0x1C000, 0x20000, 0x30000,
+                        0x800000
+                    ],
+                },
+                'findvlepages': True,
+                'mmu': [],
```

Comment:
?

Reply:
I've changed `mmu` to `vle_pages` in `vivisect/defconfig.py`